build.cake

```diff
@@ -153,6 +153,30 @@ Task("Zip-Files")
     Zip(binDir, packagePath, files);
 });
 
+Task("Create-Chocolatey-Packages")
+    .IsDependentOn("Copy-Files")
+    .WithCriteria(() => isRunningOnWindows)
+    .Does(() =>
+{
+  // Create Cake package.
+  ChocolateyPack("./nuspec/Cake.Portable.nuspec", new ChocolateyPackSettings {
+      Version = semVersion,
+      Files = new [] {
+        new ChocolateyNuSpecContent {Source = string.Concat("./../", binDir.Path.FullPath, "/Cake.exe")},
```

Comment:
As discussed, going to roll this back to the way it was for now, until above is fixed.